ledger/acctupdates.go

```diff
@@ -896,28 +926,141 @@ func (au *accountUpdates) initializeCaches(lastBalancesRound, lastestBlockRound,
 		}
 	}
 
-	for lastBalancesRound < lastestBlockRound {
-		next := lastBalancesRound + 1
+	skipAccountCacheMessage := make(chan struct{})
+	writeAccountCacheMessageCompleted := make(chan struct{})
+	defer func() {
+		close(skipAccountCacheMessage)
+		select {
+		case <-writeAccountCacheMessageCompleted:
+			au.log.Infof("initializeCaches completed initializing account data caches")
+		default:
+		}
+	}()
+	go func() {
```

Comment:
please add a comment like "this goroutine logs a message once if the parent function have not completed in `initializingAccountCachesMessageTimeout` seconds